src/scripting/lua_team.cpp

```diff
@@ -80,7 +80,7 @@ static int impl_side_get(lua_State *L)
 	return_string_attrib("faction", t.faction());
 	return_tstring_attrib("faction_name", t.faction_name());
 	return_string_attrib("color", t.color());
-	return_cstring_attrib("controller", t.controller().to_string().c_str());
+	return_cstring_attrib("controller", side_controller::get_string(t.controller()).c_str());
```

Comment:
A question to possibly defer answering until a different PR: could this use `return_string_attrib` and omit the `.c_str()`? This is one of only two callers of `return_cstring_attrib`.

Reply:
I'd guess probably, but I don't know for sure. Maybe @CelticMinstrel would know though.

Reply:
Yeah, I think that should work.

Reply:
Alright, changed then.